lib/chef/knife/ec2_server_create.rb

```diff
@@ -1537,12 +1537,13 @@ def windows_password
         if not locate_config_value(:winrm_password)
           if locate_config_value(:identity_file)
             print "\n#{ui.color("Waiting for Windows Admin password to be available", :magenta)}"
-            print(".") until check_windows_password_available(@server.id) {
+            print(".\n") unless @server
```

Comment:
Isn't there supposed to be a loop here? Before we were calling `check_windows_password_available` which would sleep for 10 seconds and try the password, and because that was an `until` we would keep doing it.

Now there's just a single `unless` on a print, which isn't useful. If we need to not look for the password until `@server` is valid, I would expect a `return false unless @server` in `check_windows_password_available` after the `sleep 10`.